scripts/common.sh

```diff
@@ -22,22 +22,27 @@
 # The fail() function defined below requires set -e to be enabled.
 set -e
 
+CUT="cut"
+if [[ "$OSTYPE" == "darwin"* ]]; then
+    CUT="gcut"
+fi
```

Comment:
Do we really want to do this? This introduces an assumption that `coreutils` is always installed on macos. And it is in CI, but that might not be the case for people who run tests locally. For that reason we should at least mention it in docs.

I mean, I'm not against it (it would actually be really convenient; I hate macos being different here for no reason) but so far we were careful to use portable commands in `common.sh` and scripts that depend on it (cmdline tests, ast import) so it seems a bit silly to just randomly introduce this now.

I'd recommend to solve it differently - by just switching to short args for `cut` (i.e. `-d` instead of `--delimiter`) - and separately decide whether we want a dependency on `coreutils` or not.

Reply:
Switched to short args now. Using `gcut` here seem to be indeed a bit of a overkill.